index.html

```diff
@@ -46,4 +46,238 @@ <h1 class="mola">  I'm Sayed Ramin<br />Glad to see<br />
+        </p>
+        <span class="call1">LET'S CONNECT</span>
+        <ul class="socials1">
+            <li>
+                <a href="https://twitter.com/SayedRamin78773"><img src="image/icons/cat.svg" alt="github" /> </a>
+            </li>
+            <li>
+                <a href="https://www.linkedin.com/in/sayed-ramin-sadat-7770a0270/"><img
+                        src="image/icons/linkedin.png" alt="linkedin" /></a>
+            </li>
+            <li>
+                <a href="#"><img src="image/icons/fingers.svg" alt="fingers" /></a>
+            </li>
+            <li>
+                <a href="https://github.com/sadat-Ramin"><img src="image/icons/twitter.svg" alt="twitter" /></a>
+            </li>
+            <li>
+                <a href="#"><img src="image/icons/m.svg" alt="M"/></a>
+            </li>
+        </ul>
+    </div>
+            <div class="btn3">
+              <button type="button" class="btn-get">Get my resume</button>
+                <ul class="accordion">
+                  <li>
+                    <div class="lan">
+                      <a class="title-link" href="#"
+                        ><span class="Language">Languages</span><i class="fa-solid fa-chevron-down solid solids"></i></p></a
+                      >
+                    </div>
+                    <ul>
+                      <li class="skill">
+                        <img class="skillـimage" src="image/works/Ellipse 1.png" alt="" />
+                        <span class="skillـname">Javascript</span>
+                      </li>
+                      <li class="skill">
+                        <img class="skillـimage" src="image/works/Ellipse 1 (1).png" alt="" />
+                        <span class="skillـname">HTML</span>
+                      </li>
+                      <li class="skill">
+                        <img class="skill_image" src="image/works/Ellipse 1 (2).png" alt="" />
+                        <span class="skillـname">CSS</span>
+                      </li>
+                    </ul>
+          
+                    <div class="frameworks">
+                      <a class="title-link" href="#">
+                        <span class="skills-title-">Frameworks</span><i class="fa-solid fa-chevron-down solid solids"></i></p></a>
+                        <ul><li class="inm"><img class="inm2" src="image/works/Separator Bottom.png"></li></ul>
+                      </div>
+                  <div class="sky">
+                      <a class="title-link" href="#">
+                        <span class="skills-title-">Skills</span><i class="fa-solid fa-chevron-down solid solids"></i></p></a
+                      >
+                      <ul><li class="inm"><img class="inm2" src="image/works/Separator Bottom.png"></li></ul>
+                  </div>
+                    </div>
+                  </li>
+                </li>
+              </ul>
+            </section>
```

Comment:
- [ ] As the previous reviewer suggested, Kindly try to match your design to the Figma design. Kindly give proper margins and spacing to match the design. :+1: 

Your design             |  Figma Design
:-------------------------:|:-------------------------:
![about myself](https://github.com/sadat-Ramin/My-portfolio/assets/70483835/31b152bc-41ff-4396-b81b-b0d203f57f5e) | ![figma](https://github.com/sadat-Ramin/My-portfolio/assets/70483835/cbfa645b-9afe-454f-b23b-560149dc4d27)